File: Bars.Gkh.Overhaul.Nso/libs/B4/model/RealEstateTypeCommonParam.js
Ext.define('B4.model.RealEstateTypeCommonParam', {
    extend: 'B4.base.Model',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'RealEstateTypeCommonParam'
    },
    fields: [
        { name: 'Id', type: 'number', defaultValue: 0 },
        { name: 'RealEstateType' },
        { name: 'CommonParamCode' },
        { name: 'Min' },
        { name: 'Max' }
    ]
});

File: Bars.Gkh/Entities/RealEstateType/RealEstateTypeCommonParam.cs
namespace Bars.Gkh.Entities.RealEstateType
{
    using Bars.Gkh.Entities;

    using Dicts;

    /// <summary>
    /// Общий параметр типа жилых домов
    /// </summary>
    public class RealEstateTypeCommonParam: BaseImportableEntity
    {
        /// <summary>
        /// Тип домов
        /// </summary>
        public virtual RealEstateType RealEstateType { get; set; }

        /// <summary>
        /// Код общего параметра
        /// </summary>
        public virtual string CommonParamCode { get; set; }

        /// <summary>
        /// Минимальное значение
        /// </summary>
        public virtual string Min { get; set; }

        /// <summary>
        /// Максимальное значение
        /// </summary>
        public virtual string Max { get; set; }
    }
}